regression/__snapshots__/validateExamplesRegression_latestOnly.test.ts.snap

```diff
@@ -160,8 +160,8 @@ Array [
     "code": "INVALID_FORMAT",
     "exampleJsonPath": "$parameters.serviceMemberId",
     "examplePosition": Object {
-      "column": 17,
-      "line": 2,
+      "column": 24,
```

Comment:
Did you sample check if column/line no are correct?

Reply:
I checked the first few and they are all correct